### PR TITLE
Added timeline to home page

### PR DESCRIPTION
**Pull request by joetho786**

### CODE REVIEW

**Comment by Sahil1479** [comment]
@joetho786 Please check the code formatting. Run flake8 command for this.

**Comment by Sahil1479** [comment]
@joetho786 Also, You should pull all recent changes before making PR and don't blindly copy-paste code from your other local copy. Like in current PR, I can see in base.html you have removed the **[Schedule](https://github.com/Sahil1479/Prometeo/pull/3/files#diff-690b81fad8df2a1f1ce37c846641d9247e3472d244f17039d2910a5ed0c98d5eL83)** navbar option added by me. Please check the changes you have made and correct them ASAP.

File: templates/base.html [hunk 1]
```diff
@@ -210,15 +196,9 @@ <h4>Our Social Networks</h4>
 
     <!-- Template Main JS File -->
     <script src="{% static 'js/main.js' %}"></script>
-    <script>
```
Comment (Sahil1479) [inline templates/base.html:213]: You have also removed this AOS script. Please carefully check all the files.

Comment (Sahil1479) [reply templates/base.html:213]: @joetho786

Comment (joetho786) [reply templates/base.html:213]: ok will do it asap

File: templates/home.html [hunk 2]
```diff
@@ -11,6 +11,234 @@
 {% block style %}
 <link href="{% static 'css/home.css' %}" rel="stylesheet">
 <link href="{% static 'css/particle.css' %}" rel="stylesheet">
+<style>
```
Comment (Sahil1479) [inline templates/home.html:14]: @joetho786 Avoid writing CSS in a style tag, create a separate CSS file.

**Pull request merged**